docs/pvc-recovery.md

```diff
@@ -79,10 +79,10 @@ To begin, this guide assumes you have a currently running cluster in the `opensh
         truststore:
           path: /etc/elasticsearch/secret/truststore
           password: tspass
-          type: jks
+          type: pkcs12
         keystore:
           path: /etc/elasticsearch/secret/key
```

Comment:
this file being created is currently in the `jks` format. i think we'd need to use its `.p12` equivalent

you'd have to look through here [1] (or just look on a running es node) to see which files those are

[1] https://github.com/openshift/origin-aggregated-logging/blob/master/elasticsearch/utils/logging#L171-L177

Reply:
Thanks. I've made the changes.

Reply:
Will this have an impact ont hsi? https://github.com/openshift/origin-aggregated-logging/blob/c41591296ba3e69ad1bd9ce9e53b2ce01db21835/elasticsearch/utils/logging

Reply:
unfortunately i think so.. we may need to do this in three parts..
1) add logic to have a trust store chain created in that file (to go along side the jks truststores)
2) update EO to use the pkcs12 format files instead of jks
3) remove the jks gen (this can be a follow up removal and isn't really necessary but would be good to do)

Reply:
Sorry, I'm a bit lost.

Doesn't this PR do 2 and the one I made in the OAL do both 1 and 3? Or are you saying break up the OAL PR into two PRs. One that essentially is sorta a copy paste of the scenario to do make both jks and p12 formats and another to remove the jks entirely?

OAL PR: https://github.com/openshift/origin-aggregated-logging/pull/2077